README.md

```diff
@@ -38,7 +39,9 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```
+Scope refers to the context of a variable or function.  There is a local scope and a global one.  If a variable
```

Comment:
:+1: This is correct, but for technical accuracy, I'd use the term `defined` rather than `referenced`.